src/index.css

```diff
@@ -630,6 +630,28 @@ small {
   color: #2FA4E7;
 }
 
+.cell-sort .sort-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.cell-sort .sort-container .sort-icon {
+  font-size: 20px;
+  opacity: 0.4;
+}
+
+.cell-sort .sort-container .sort-icon.active {
+  opacity: 1;
+}
+
+.cell-sort .sort-container .sort-icon.sort-icon-up {
+  transform: translateY(7px);
+}
+
+.cell-sort .sort-container .sort-icon.sort-icon-down {
+  transform: translateY(-7px);
+}
+
```

Comment:
Can we move this to an inline style object in `SimpleTable.js`? We have been trying to move away from using index.css for a few different reasons, chief among them being the difficulty in refactoring styles when they end up used across many components.

Reply:
All decisions are always open for discussion! Feel free to ignore this suggestion and we can schedule a conversation with the larger team.

Reply:
Ok, if you don't mind then, I'll leave the styles in place! The reasoning for that being that the cell-sort styles were already in there, there are other parts of the application using them, and it might be weird / unintuitive to have sort styling in multiple places.